woocommerce/woocommerce
# Update `woocommerce-admin` documentation

### All Submissions:






### Changes proposed in this Pull Request:



Closes #32416.

- Update `woocommerce-admin` README.md and [Set up development environment](https://github.com/woocommerce/woocommerce/wiki/How-to-set-up-WooCommerce-development-environment) wiki 
- Update `./docs/*`
- Fix broken example webpack config
- Remove building docs storybook command and `./docs/components`

### How to test the changes in this Pull Request:

1. Review the changes and the [wiki page](https://github.com/woocommerce/woocommerce/wiki/How-to-set-up-WooCommerce-development-environment)
2. Should be able to build an example plugin: `WC_EXT=add-navigation-items pnpm nx example woocommerce-admin`.

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

Docs: Update `woocommerce-admin` documentation

### FOR PR REVIEWER ONLY:
